proposals/adrianhopebailie-201908.md

```diff
@@ -562,6 +586,106 @@ In the response the CNP can provide a value date that they commit to, based on
 their knowledge of and SLAs with the external systems that will receive the
 payment.
 
+
+### Generalizing Transaction Outcome Notifications
+
+`POST /quotes#subscribers` is a list of `Subscriber` objects each
+representing an interested party in the transaction.
+
+The `Subscriber` object takes the form of:
+
+| Name               | Cardinality | Type               | Description                                                                             |
+| ------------------ | ----------- | ------------------ | --------------------------------------------------------------------------------------- |
+| id                 | 1           | `FspId`            | The id of a participant interested in the transaction                                   |
+| role               | 1           | `SubscriptionRole` | The role of the subscription                                                            |
+
+Where `SubscriptionRole` is a enum of the following values:
+
+| Name               | Description                                | 
+| ------------------ | ------------------------------------------ | 
+| `PAYER`            | The sender of the funds                    | 
+| `PAYEE`            | The recipient of the funds                 | 
+| `INITIATOR`        | The participant initiating the transaction | 
```

Comment:
Hi Lewis, in the API Definition docs, I think especially Patters, Use Cases, the Initiator term is used to identify entities / FSPs that initiate several use cases.. Taking that into consideration, can we use something like "PISP" to be more specific but also avoid ambiguity?

Reply:
Yeah that's a good point. Thanks @elnyry-sam-k

Reply:
Fixed